Classes/Command/DeutscherWetterdienstCommand.php

```diff
@@ -0,0 +1,352 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the package jweiland/weather2.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace JWeiland\Weather2\Command;
+
+use Doctrine\DBAL\Exception;
+use JWeiland\Weather2\Domain\Model\DwdWarnCell;
+use JWeiland\Weather2\Domain\Repository\DwdWarnCellRepository;
+use JWeiland\Weather2\Utility\WeatherUtility;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Log\LoggerInterface;
+use Psr\Log\LogLevel;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use TYPO3\CMS\Core\Authentication\BackendUserAuthentication;
+use TYPO3\CMS\Core\Database\ConnectionPool;
+use TYPO3\CMS\Core\Http\RequestFactory;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\CMS\Extbase\Persistence\Generic\PersistenceManager;
+use TYPO3\CMS\Extbase\Service\CacheService;
+
+final class DeutscherWetterdienstCommand extends Command
+{
+    public const API_URL = 'https://www.dwd.de/DWD/warnungen/warnapp/json/warnings.json';
+    protected string $dbExtTable = 'tx_weather2_domain_model_weatheralert';
+
+    /**
+     * JSON response from dwd api
+     *
+     * @var array<string, mixed>
+     */
+    protected array $decodedResponse = [];
+
+    /**
+     * @var array<string, mixed>
+     */
+    protected array $keepRecords = [];
+
+    /**
+     * @var array<string, mixed>
+     */
+    protected array $warnCellRecords = [];
+
+    public function __construct(
+        protected readonly PersistenceManager $persistenceManager,
+        protected readonly DwdWarnCellRepository $dwdWarnCellRepository,
+        protected readonly RequestFactory $requestFactory,
+        protected readonly LoggerInterface $logger,
+        protected readonly ConnectionPool $connectionPool,
+        protected readonly CacheService $cacheService,
+    ) {
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->setDescription('Fetch and process weather alerts from Deutscher Wetterdienst')
+            ->setHelp('Calls the Deutscher Wetterdienst api and saves response in weather2 format into database')
+            ->addArgument(
+                'selectedWarnCells',
```

Comment:
Just "warnCells". It was a selectbox in scheduler tasks before, but not in command.